publiccode.yml

```diff
@@ -0,0 +1,79 @@
+publiccodeYmlVersion: "0.4"
+
+name: Casual
+applicationSuite: #optional
+url: "https://github.com/casualcore/casual.git"
+landingUrl: "https://casual-software.com/" #optional
+softwareVersion: "" #optional
+releaseDate: "" #optional
+logo: "resources/casual-box-left.png" #optional preference svg.
+platforms:
+  - linux
+
+categories: # ??? restricted list. https://yml.publiccode.tools/categories-list.html
+  - it-development
+
+
+indendedAudience: #optional ??? restricted list: https://yml.publiccode.tools/scope-list.html#valid-tags
+  scope:
+    - welfare
+
+usedBy: #optional
+  - Försäkringskassan
+  - Bolagsverket
```

Comment:
```
 - Swedish social insurance system
 - Swedish Companies Registration Office
 ```

Reply:
Social Insurance Agency i have seen before in the FK site in english.
https://www.forsakringskassan.se/english/moving-to-working-studying-or-newly-arrived-in-sweden/social-insurance-in-sweden
Also, should we have the actual names in brackets a bit like in the links on the side of that site?
```yaml
- Swedish Social Insurance Agency (Försäkringskassan)
- Swedish Companies Registration Office (Bolagsverket)
```

Reply:
I found it here: https://www.forsakringskassan.se/english

Reply:
I see. But that is the system, rather than the company perhaps, on the previous link i provided in the first paragraph it seemed to describe it like that too.
Bit more digging... most things are agencies from what I can see.
https://www.government.se/government-agencies/social-insurance-agency--forsakringskassan/
https://www.government.se/government-agencies/swedish-pensions-agency-pensionsmyndigheten/
Though bolagsverket is an office :)
https://www.government.se/government-agencies/swedish-companies-registration-office/

Reply:
Maybe there's a distinction since they're self financed.